### PR TITLE
Updates to assignment wizard

### PR DESCRIPTION
**Pull request by lwjameson**
## In this PR

- Updates to assignment wizard to allow selection of read only layers.
- Misc fixes.

### CODE REVIEW

File: src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx [hunk 1]
```diff
@@ -35,127 +38,132 @@ interface TeamProps {
...
+        ...members,
+        ...group.members
+          .map((m) => m.user)
+          .filter((u) => selected.includes(u.id)),
+      ],
+      [] as UserProfile[]
+    );
 
     props.onChange(members);
   }, [selected, props.project.groups, props.onChange]);
```
Comment (rsimon) [inline src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx:115]: I noticed that this component drops into an infinite re-render loop. (This is probably an error that I introduced in the first place :-) Anyways: `props.onChange` shouldn't be in the dependency array in this case.

**Comment by rsimon** [comment]
Looks good overall! I got some weird errors though. Does this already work against the COVE staging backend? In my case, right after I created an assignment, the preflight HTTP call to the `create_context_rpc` went well, but then the actual POST request got an HTTP 404.

**Review by rsimon — APPROVED** [review]

File: src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx [hunk 2]
```diff
@@ -35,127 +38,132 @@ interface TeamProps {
...
-      const { field } = sorting;
-      const order = sorting.ascending ? -1 : 1;
-      return (a[field] < b[field]) ? -order : (a[field] > b[field]) ? order : 0;
-    } else {
-      return 0;
-    }
-  });
+const getMembers = (groups: Group[], sorting?: Sorting): Member[] =>
+  groups
+    .filter((g) => !g.is_admin)
```
[the remark this answers is not in the document]
Comment (lwjameson) [reply src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx:64]: Well they really do not need to be added.  They already have access and cannot be blocked by not being on the team.  Maybe we list them but do not allow removing?  Or put in a note that says that all project admins have access?

[the remark this answers is not in the document]
Comment (rsimon) [reply src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx:64]: +1 on listing them, in a kind of "disabled" mode. This way, the user is aware what's going on.

**Pull request merged**